[Gateway] Block page and TLS decryption cert limitations (#18977)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/policies/gateway/block-page.mdx

| User ID      | The ID of the user who visited the page. Currently, User IDs are not surfaced in the dashboard and can only be viewed by calling the [API](/api/resources/zero_trust/subresources/access/subresources/users/methods/list/). |
| Device ID    | The ID of the device that visited the page. This is generated by the WARP client.                                                                                                                                           |
| Block Reason | Your policy-specific block message.                                                                                                                                                                                         |

## Limitations

If your users receive a security risk warning in their browser when visiting a blocked page, check that you have correctly [installed a Cloudflare certificate](/cloudflare-one/connections/connect-devices/user-side-certificates/manual-deployment/) on their devices.

If an HTTP request that matches a block policy does not arrive at the same Cloudflare data center as its DNS query, Gateway will display the default block page instead of your custom block page.

> If the HTTP request comes from a different IP address than the DNS request, Gateway may not display the rule ID, custom message, and some other fields on the block page. This can happen when a recursive DNS resolver's source IP address differs from the user device's IP address.

Author: ```suggestion
If the HTTP request comes from a different IP address than the DNS request, Gateway may not display the rule ID, custom message, or other fields on the block page. This can happen when a recursive DNS resolver's source IP address differs from the user device's IP address.
```
